src/actions.ts

```diff
@@ -190,6 +190,7 @@ export default class ActionCreator extends BaseActionCreator {
   static readonly RESET_ADOBE_ID = "RESET_ADOBE_ID";
 
   static readonly DIAGNOSTICS = "DIAGNOSTICS";
+  static readonly QUICKSIGHT_EMBEDDED_URL: "QUICKSIGHT_EMBEDDED_URL";
```

Comment:
I think I missed this before, but the value of this constant needs to assigned with and equal sign.
```typescript
static readonly QUICKSIGHT_EMBEDDED_URL = "QUICKSIGHT_EMBEDDED_URL";
```

This might be why we were running into trouble using Redux actions dispatch. I also note that the various state constants for the `QUICKSIGHT_EMBEDDED_URL` action are not defined, which would also prevent actions dispatch from working properly. There are assumptions/naming conventions for these on which `fetchJSON` depends. A good template for that would be the `STATS` action:
```typescript
  ...
  static readonly STATS = "STATS";
  ...
  static readonly STATS_REQUEST = "STATS_REQUEST";
  static readonly STATS_SUCCESS = "STATS_SUCCESS";
  static readonly STATS_FAILURE = "STATS_FAILURE";
  static readonly STATS_LOAD = "STATS_LOAD";
  ...
```

Reply:
I'll fix.

Reply:
Good catch.

Reply:
I'm understanding what you did with the Stats piece now a little better.   I'd like to get this out there for front-end feedback, but I will plan to rework the fetching using RTK before we goes live.